documentation.md

```diff
@@ -0,0 +1,155 @@
+
+### `backup`
+|     |     |
+| --- | --- | 
+| Description | Causes ScamGuard to backup and archive its internal database outside of its normal scheduled time |
+| Restricted to | MaintainerRole |
+
+### `forceleave server`
+|     |     |
+| --- | --- | 
+| Description | Forces ScamGuard to leave |
+| Cooldown | once every 3 seconds |
+| Restricted to | MaintainerRole |
+
+| Arguments | Type |
+| --- | --- |
+| server |  discord id of a server |
+
+### `forceactivate server`
+|     |     |
+| --- | --- | 
+| Description | Forces ScamGuard activate on a server |
+| Restricted to | MaintainerRole |
+
+| Arguments | Type |
+| --- | --- |
+| server | discord id of a server |
+
+### `retryactions server count`
+|     |     |
+| --- | --- | 
+| Description | Forces ScamGuard to retry a set of actions on a server |
+| Restricted to | MaintainerRole |
+
+| Arguments | Type |
+| --- | --- |
+| server |  discord id of a server |
+| count | count of last actions to retry |
+
+### `retryinstance instance count`
+|     |     |
+| --- | --- | 
+| Description | Forces ScamGuard to retry a set of actions for a instance of ScamGuard |
+| Restricted to | MaintainerRole |
+
+| Arguments | Type |
+| --- | --- |
+| instance | id of a ScamGuard instance |
+| count | count of last actions to retry |
+
+### `ping instance`
+|     |     |
+| --- | --- | 
+| Description | Pings an instance of ScamGuard |
+| Restricted to | MaintainerRole |
+
+| Arguments | Type |
+| --- | --- |
+| instance | id of a ScamGuard instance |
+
+### `print`
+|     |     |
+| --- | --- | 
+| Description | Prints stats and information about all connected ScamGuard instances |
+| Restricted to | MaintainerRole |
+
+### `scamban userid`
+|     |     |
+| --- | --- | 
+| Description | Bans a user id |
+| Restricted to | MaintainerRole |
+
+| Arguments | Type |
+| --- | --- |
+| userid | discord userid to ban |
+
+### `scamunban userid`
+|     |     |
+| --- | --- | 
+| Description | Unbans a user id |
+| Restricted to | MaintainerRole |
+
+| Arguments | Type |
+| --- | --- |
+| userid | discord userid to unban |
+
+### `activate`
+|     |     |
+| --- | --- | 
+| Description | Activates all servers and brings in previous bans if caller owns any known servers |
+
+### `deactivate`
+|     |     |
+| --- | --- | 
+| Description | Deactivates all servers and prevents any future ban information from being shared if the caller owns any known servers |
+
+### `scamcheck userid`
+|     |     |
+| --- | --- | 
+| Description | Checks to see if a userid is banned |
+| Cooldown | once every 3 seconds |
+
+| Arguments | Type |
+| --- | --- |
+| userid | discord userid to check |
+
+
+## Global
+> These commands can be run on any server where ScamGuard is joined
+
+### `check userid`
+|     |     |
+| --- | --- | 
+| Description | Checks to see if a userid is banned |
+| Permissions | can ban members |
+
+| Arguments | Type |
+| --- | --- |
+| userid | discord userid to check |
+
```

Comment:
Missing the commands for reporting.

Reply:
I genuinely have no idea how i missed these. updated.